packages/cli/configs/ttk/basic/csharp/TeamsApp/README.md

```diff
@@ -0,0 +1,22 @@
+# Welcome to Teams Toolkit!
+
+## Quick Start
+
+1. In the debug dropdown menu, select Dev Tunnels > Create A Tunnel (set authentication type to Public) or select an existing public dev tunnel
+</br>![image](https://raw.githubusercontent.com/OfficeDev/TeamsFx/dev/docs/images/visualstudio/debug/create-devtunnel-button.png)
+2. Right-click the 'TeamsApp' project in Solution Explorer and select **Teams Toolkit > Select Microsoft 365 Account**
+3. Sign in to Teams Toolkit with a **Microsoft 365 work or school account**
```

Comment:
It's probably good to change this to M365 Agents Toolkit as well

Reply:
will make these fixes as a fast follow